app.controllers.js

```diff
@@ -76,4 +76,15 @@ exports.updateVotes = (req, res, next) => {
   .catch((err) => {
     next(err)
   })
+}
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params
+  return findComment(comment_id)
+  .then(() => {
+    res.status(204).send()
+  })
+  .catch((err) => {
+    next(err)
+  })
```

Comment:
Nice, we dont need to return findComment here but otherwise good logic here. As an alternative to having an empty send() here you can use `res.sendStatus(204)` which can look a little nicer for this.